docs/built-in-functions.rst

```diff
@@ -368,7 +368,7 @@ Data Manipulation
 
     * ``b``: value being sliced
     * ``start``: start position of the slice
-    * ``length``: length of the slice
+    * ``length``: length of the slice, must be constant. Immutable variable doesn't supported.
```

Comment:
Grammar nit:
`Immutables and variables are not supported`

Reply:
```suggestion
    * ``length``: length of the slice, must be constant. Immutables and variables are not supported.
```